src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap

```diff
@@ -38,6 +46,11 @@ exports[`Story Page should not show the pop-out timestamp when allowDateStamp is
 
 @media (min-width:15rem) and (max-width:24.9375rem) {
     .emotion-662 {
+<<<<<<< HEAD
+      width: initial;
+      margin: 0;
+=======
+>>>>>>> eb919405efd3e56cab31295ce389b578276301d5
```

Comment:
the snapshots have conflict artefacts in them here, probably best to delete the file and recreate the snapshots, I'd expect no differences to the snapshots as you haven't changed the tests